recipe/meta.yaml

```
@@ -40,11 +34,12 @@ requirements:
     - streamlit-camera-input-live >=0.2.0
     - streamlit-vertical-slider >=1.0.2
     - streamlit-toggle-switch >=1.0.2
-    - htbuilder 0.6.1
+    - htbuilder
```

This has actually increased

https://github.com/arnaudmiribel/streamlit-extras/blob/28c9d995c9b4a3ca04484fa4b3bb9e34855149f6/pyproject.toml#L29C14-L29C21
```suggestion
    - htbuilder >=0.6.2
```

---

Added the pinning.